test/edu-benefits/definitions/personId.unit.spec.jsx

```diff
@@ -9,7 +9,7 @@ import * as personId from '../../../src/js/edu-benefits/definitions/personId';
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
 describe('Edu personId', () => {
-  const uiSchema = personId.uiSchema();
+  const uiSchema = personId.uiSchema('veteran', 'view:noSSN');
```

Comment:
It looks like the preference is to explicitly provide inputs for invocations of `personId.uiSchema` rather than to rely on default values. Should this be changed throughout?